SU-HSDO/stanford_mrc
# Mrcd8 213 media ux

# READY FOR REVIEW

# Summary
- Alot of UX improvements for the media browser

# Needed By (Date)
- End of sprint

# Urgency
- High

# Steps to Test

1. Checkout branch
2. update database & clear cache
3. Create any content with a wysiwyg
4. open the media browser and confirm the following:
  - limits to only 1 item chosen in the library
  - Choosing an item shows a tray at the bottom that can be hidden via the button on the right
  - on the embed a file: the upload field goes away after a file is added
  - on the embed a file: a thumbnail should be on the right.
  - on the embed a video: the field should be a text field and a button below it
  - on the embed a video: a thumbnail should be on the right after adding
  - after adding a video or file a message at the top indicates the item has been added to the library.
5. choose an image to embed and confirm the following
  - on the second form where you select an image style, there the title text field should only display if the link field is populated
  - on that form ^ alot of help text should be there now
6. also confirm that if you upload a file, but then switch to the media library, the uploaded file exists in the media library but is no longer "selected" to continue.



# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
